development/docker-compose/docker-compose.yaml

```diff
@@ -16,18 +17,17 @@ x-common-variables: &common-variables
   eureka.instance.initialStatus: STARTING
   management.endpoints.web.exposure.include: '*'
   management.endpoints.enabled-by-default: true
-  management.security.roles: ANONYMOUS,ACTUATOR,USER
+  management.security.roles: ANONYMOUS,ACTUATOR
   management.endpoint.env.show-values: ALWAYS
+  management.endpoint.env.post.enabled: true
   management.endpoint.configprops.show-values: ALWAYS
   # Observability - Tracing and Prometheus Configuration
   management.tracing.enabled: false
-  management.endpoint.prometheus.enabled: true
```

Comment:
this is redundant since we already have `management.endpoints.enabled-by-default: true`